components/badge.css

```diff
@@ -0,0 +1,62 @@
+:root{
+    --badge-bg-color: #b3d2f5;
+    --discord-color:#738ADB;
+    --messanger-color:#006AFF;
+}
+
+#badge-container p span{
+    background-color: var(--badge-bg-color);
+    border-radius: 5px;
+    padding: 2px;
+}
+
+.icons{
+    display: flex;
+    gap: 2.5rem;
+    padding: 1rem 2rem;
+}
+
+.icon i{
+    font-size: 3.5rem;
+}
+
+.icon span{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 1rem;
+    width: 1rem;
+    padding: 12px;
+    background-color: red;
+    color: white;
+    border-radius: 50%;
+    position: relative;
+    bottom: 4rem;
+    left: 2.5rem;
+}
+
+.discord-icon {
```

Comment:
instead of discord icon the name can be more generic because you never know what icon the user will use